api/api.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
+
+package api;
+
+option go_package = "github.com/ligato/vpp-agent/api";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+option (gogoproto.messagename_all) = true;
+
+import "google/protobuf/any.proto";
+
+message ModelSpec {
+    string module = 1;
+    string class = 2;
+    string version = 3;
+    string kind = 4;
+}
+
+message Model {
+    google.protobuf.Any any = 1;
+    map<string,string> metadata = 2;
+}
+
+service SyncService {
+    rpc Sync (SyncRequest) returns (SyncResponse) {};
+    rpc Obtain (ObtainRequest) returns (ObtainResponse) {};
```

Comment:
Semantically, what is 'Obtain' and what is 'Sync' ?  Most devs don't think in quite these terms, so might be useful to put a very fine point on what we mean by them.

Reply:
So Sync is 'Set' and 'Obtain' is 'Get' ?  Those might be clearer :)

Reply:
To me, GET/SET is used for things which are equal for both operations, meaning that via **GET** you get the exact value the was set via **SET**. Which is not the case here, so I tried to use different wording. However I have to agree that I'm not very fond of this exact wording to be honest.

Perhaps, **Configure**/**Retrieve** are more suitable candidates here.

Reply:
Or perhaps just Read/Write?